console/src/main/webapp/manager/app/app.es6

```diff
@@ -145,6 +145,18 @@ angular.module('manager', [
     moment.locale($translate.use())
     return date => moment(date).format('LL')
   }])
+  .directive('shortname', () => ({
+    require: 'ngModel',
+    restrict: 'A',
+    link: (scope, elm, attrs, ctrl) => {
+      const alphanum = v => v && v.match(/^\w*$/)
```

Comment:
also need to allow the underscore char (ex: role `EL_TRUITE`)

Reply:
https://developer.mozilla.org/fr/docs/Web/JavaScript/Guide/Expressions_r%C3%A9guli%C3%A8res#special-word :sunglasses: